topeft/modules/corrections.py

```diff
@@ -609,11 +835,14 @@ def AttachTauSF(events, taus, year, vsJetWP="Loose"):
 
 def AttachPerLeptonFR(leps, flavor, year):
     # Get the flip rates lookup object
+    if year not in clib_year_map.keys():
+        raise Exception(f"Error: Unknown year \"{year}\"\n"".")
+
     if year == "2016APV": flip_year_name = "UL16APV"
     elif year == "2016": flip_year_name = "UL16"
     elif year == "2017": flip_year_name = "UL17"
     elif year == "2018": flip_year_name = "UL18"
-    else: raise Exception(f"Not a known year: {year}")
+    else: flip_year_name = "UL18" #TO READAPT when fakefactors are ready #raise Exception(f"Not a known year: {year}")
```

Comment:
Is this a temporary workaround for Run 3? Maybe put the Exception back in (commented out) with a note and/or a GitHub issue that we need to reenable it.

Reply:
Yes, I will open a GH issue for future reference

Reply:
https://github.com/TopEFT/topeft/issues/437